.github/workflows/build_windows.yaml

```diff
@@ -105,6 +109,20 @@ jobs:
             - name: Install Deps
               run: "yarn install --frozen-lockfile"
 
+            - name: Insert config snippet
+              if: inputs.extra_config != ''
+              shell: bash
+              run: |
+                mkdir config-edit
```

Comment:
Unsure on this approach, but I couldn't think of a better way to edit config.json since by this point it's already zipped up in the asar.

Reply:
Looks sane for a temporary thing